src/js/tabs/firmware_flasher.js

```diff
@@ -18,6 +18,18 @@ TABS.firmware_flasher.initialize = function (callback) {
     var intel_hex = false; // standard intel hex in string format
     var parsed_hex = false; // parsed raw hex in array format
     var targetConfig; // the Unified Target configuration to be spliced into the configuration
+    var isConfigLocal = false; // Set to true if the user loads one locally
+    var unifiedConfig; // Unified target configuration loaded from the menu, used when throwing out a local config
+
+    // Is there a list of these elsewhere?, used in parseUnifiedBuilds and ... look for unifiedTargets[0]
+    var unifiedTargets = [ 'STM32F411', 'STM32F405', 'STM32F745', 'STM32F7X2' ];
+    self.peekTargetConfig = function () { // This maybe should get removed before merging.
+        return targetConfig;
+    }
+    var unifiedSource = 'https://api.github.com/repos/betaflight/unified-targets/contents/configs';
+    //unifiedSource = 'https://api.github.com/repos/betaflight/betaflight/contents/unified_targets/configs';
```

Comment:
Please remove commented out code, or add a comment indicating the reason it is left in.